src/Schedules.jl

```diff
@@ -114,12 +114,26 @@ struct EveryDtSchedule{T} <: AbstractSchedule
     dt::T
 
     """
-        EveryDtSchedule(dt; t_start = zero(dt))
+        EveryDtSchedule(dt; t_last = zero(dt))
 
     True every time the current time is larger than the previous time this schedule was true + dt.
```

Comment:
very minor, but for consistency, we could call "previous time" in the doc string "last time".

Reply:
Fixed